src/tests/dictionaryConcepts/container/EditConcept.test.jsx

```diff
@@ -625,4 +652,26 @@ describe('Test suite for mappings on existing concepts', () => {
     instance.updateSourceEventListener(event, url);
     expect(instance.state.mappings[1].map_type).toEqual('SAME-AS');
   });
+
+  it('should call removeCurrentAnswer', () => {
```

Comment:
What should call remove current answer? And when?

Reply:
Updated too